cross/tvheadend/Makefile

```diff
@@ -1,7 +1,7 @@
 PKG_NAME = tvheadend
-PKG_VERS = 3.4patch1
+PKG_VERS = 4.0.9
 PKG_EXT = tar.gz
-PKG_DIST_NAME = $(PKG_VERS).$(PKG_EXT)
+PKG_DIST_NAME = v$(PKG_VERS).$(PKG_EXT)
```

Comment:
Add a `PKG_DIST_FILE` here (usually along the lines of `PKG_DIST_FILE = $(PKG_NAME)-$(PKG_VERS)`. Redownload and recreate digests after that.

The result is that the file is stored in DISTRIB_DIR with a name that can be tied to a package. Same applies to the contents of the digests file.